src/OpenTelemetry.Exporter.ApplicationInsights/ApplicationInsightsTraceExporter.cs

```diff
@@ -38,6 +39,7 @@ public class ApplicationInsightsTraceExporter : SpanExporter, IDisposable
         public ApplicationInsightsTraceExporter(TelemetryConfiguration telemetryConfiguration)
         {
             this.telemetryClient = new TelemetryClient(telemetryConfiguration);
+            this.telemetryClient.Context.GetInternalContext().SdkVersion = "ot:" + GetAssemblyVersion();
```

Comment:
there will be more changes coming here (for OpenCensus we also had language here) https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/SDK-VERSIONS.md#sdk-names

It's fine to keep it, but we should understand our strategy here and whether we want to support it through current schema or do nicer less verbose things through resource API

Reply:
yes i understand. i added this as a mere starting point to help me distinguish which telemetry was coming from OT exporter.